datasets/steam/README.md

```diff
@@ -59,3 +59,33 @@ Using this dataset, we suggest you apply the following customization:
 We provide two files:
 - `steam-games.json` containing the dataset.
 - `settings.json` containing our suggested settings for this dataset.
+
+## Quick setup
+
+If you want to quickly create an index with the documents and the settings, you can go to the `setup_steam_instance` folder and do the following.
+
+Install dependencies:
```

Comment:
Since we are not a library, we can chose to use only `yarn` or `npm` the same way we do in the contributing guides of js projects. 

For consistency I would suggest we remove `npm` and keep `yarn`